application/db/storeKeyPrefix_test.go

```diff
@@ -1,23 +0,0 @@
-/*
```

Comment:
why deleting this?

Reply:
It is a commented out code. If we need it for some reason it is better to create an issue with the link to the commit when that code exists.